### PR TITLE
feat(docker): add image prune alias

### PR DESCRIPTION
**Pull request by alvincrespo**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- Adds `dipru` alias to docker plugin

## Other comments:

**Usage**

```
~ dipru
WARNING! This will remove all images without at least one container associated to them.
Are you sure you want to continue? [y/N] y
Deleted Images:
deleted: sha256:3eec49fdfa1ee1da964a3acb357213b6174d6a76cb25c322c60fb2c1305b9b00
untagged: postgres:13.4
untagged: postgres@sha256:1adb50e5c24f550a9e68457a2ce60e9e4103dfc43c3b36e98310168165b443a1
...
Total reclaimed space: 819.8MB
```

### CODE REVIEW

File: plugins/docker/docker.plugin.zsh [hunk 1]
```diff
@@ -6,6 +6,7 @@ alias dib='docker image build'
 alias dii='docker image inspect'
 alias dils='docker image ls'
 alias dipu='docker image push'
+alias dipru='docker image prune -a'
```
Comment (alvincrespo) [inline plugins/docker/docker.plugin.zsh:9]: For reference, https://docs.docker.com/reference/cli/docker/image/prune/

**Review by carlosala — APPROVED** [review]
LGTM!

**Comment by carlosala** [comment]
Thanks for your contribution!

**Comment by alvincrespo** [comment]
@carlosala Thanks so much for the merge and also adding this to the docs, I should have done that.

**Comment by jedrekdomanski** [comment]
Thanks!

**Pull request merged**